src/viam/sdk/common/client_helper.hpp

```diff
@@ -16,7 +17,7 @@ class ClientHelper {
     static void default_rsc_(RequestType&) {}
     static void default_rhc_(const ResponseType&) {}
     static void default_ehc_(const ::grpc::Status& status) {
-        throw std::runtime_error(status.error_message());
+        throw GRPCException(status.error_message(), status.error_code());
```

Comment:
I'm curious what to do about `@throws` doc comments. Any method that uses the client helper with the `default_ehc_` can now feasibly throw a `GRPCException` here (they all _were_ able to throw a `std::runtime_error` before we just didn't document it). Should I document that on all client functions? How important is `@throws`?

Reply:
Sounds good; there are some `throws` doc comments I should still fix, but I'll handle that after we agree on the general structure (codes vs. conditions vs. child classes).